security/rbac/delta_configs/server.properties.delta

```diff
@@ -43,3 +43,8 @@ listener.name.rbac.oauthbearer.sasl.jaas.config= \
 listener.name.rbac.oauthbearer.sasl.server.callback.handler.class=io.confluent.kafka.server.plugins.auth.token.TokenBearerValidatorCallbackHandler
 # Set SASL callback handler for handling tokens on login. This is essentially a noop if not used for inter-broker communication.
 listener.name.rbac.oauthbearer.sasl.login.callback.handler.class=io.confluent.kafka.server.plugins.auth.token.TokenBearerServerLoginCallbackHandler
+
+# Setup MetricsReporter (point at PLAINTEXT port)
+metric.reporters=io.confluent.metrics.reporter.ConfluentMetricsReporter
+confluent.metrics.reporter.bootstrap.servers=localhost:9093
+confluent.metrics.reporter.topic.replicas=1
```

Comment:
These lines are unnecessary -- they are added automatically by Confluent CLI

https://github.com/confluentinc/cli/blob/4010230fae1d5f5038a31d64a792ba2dd08dde61/cp_cli/confluent.sh#L893-L912

Results in

```
metric.reporters=io.confluent.metrics.reporter.ConfluentMetricsReporter
confluent.metrics.reporter.bootstrap.servers=localhost:9093
confluent.metrics.reporter.topic.replicas=1
```

Reply:
@ybyzek 

well it seems like it would default to `confluent.metrics.reporter.bootstrap.servers=localhost:9092` not `9093` right?

so maybe just include this one line:
```
confluent.metrics.reporter.bootstrap.servers=localhost:9093
```
?

Reply:
The way the demo is setup, it will pick 9093, so this line is still redundant.

https://github.com/confluentinc/cli/blob/4010230fae1d5f5038a31d64a792ba2dd08dde61/cp_cli/confluent.sh#L300
https://github.com/confluentinc/examples/blob/5.3.0-post/security/rbac/delta_configs/server.properties.delta#L29